src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,413 @@
+
+#### Background reading
+
+[EIP-1559 (Robust Incentives Group)](https://ethereum.github.io/abm1559/notebooks/eip1559.html)
+
+#### Recent research
+
+[EIP-4396](https://eips.ethereum.org/EIPS/eip-4396)
+
+[Multidimensional EIP-1559](https://ethresear.ch/t/multidimensional-eip-1559/11651)
+
+[Making EIP-1559 more like an AMM curve](https://ethresear.ch/t/make-eip-1559-more-like-an-amm-curve/9082)
+
+[Cross domain MEV](http://arxiv.org/abs/2112.01472)
+
+
+## Clients and protocol research
+
+### Consensus clients
+
+The consensus layer is concerned with the proof-of-stake mechanism securing Ethereum from the merge onwards. There is abundant ongoing research and development relating to PoS concepts and implementation, for example identifying and patching vulnerabilities, quantifying cryptoeconomic security, increasing the security or performance of client implementations, building light clients and developing staking apps. Some fundamental redesigns of the protocol to confer large improvements to Ethereum are also being researched, for example single slot finality. Furthermore, the efficiency and safety and monitoring of the peer-to-peer networking between consensus clients are also important research topics.
+
+#### Background reading
+
+[Casper-FFG paper](https://arxiv.org/abs/1710.09437)
+
+[Casper-FFG Explainer](https://arxiv.org/abs/1710.09437)
+
+[Gasper paper](https://arxiv.org/abs/2003.03052)
+
+
+##### Client implementations
+
+[Prysm client](https://docs.prylabs.network/docs/how-prysm-works/beacon-node/)
+
+[Lighthouse client](https://lighthouse-book.sigmaprime.io/)
+
+[Lodestar client](https://lodestar.chainsafe.io/)
+
+[Teku client](https://consensys.net/knowledge-base/ethereum-2/teku/)
+
+[Nimbus client](https://nimbus.team/)
+
+
+#### Recent research
+
+[Ethresear.ch Consensus](https://ethresear.ch/c/consensus/29)
+
+[Availability/Finality dilemma](https://arxiv.org/abs/2009.04987)
+
+[Single slot finality](https://ethresear.ch/t/a-model-for-cumulative-committee-based-finality/10259)
+
+[Proposer-builder separation](https://notes.ethereum.org/@vbuterin/pbs_censorship_resistance)
+
+
+### Execution clients
+
+The execution layer is concerned with executing transactions, running the EVM and generating execution payloads to pass to the consensus layer. There are many active areas of research, especially as execution clients are updated ready for the merge. This includes building out light client support, research into gas limits and incorporating new data structures (e.g. Verkle Tries). 
+
+#### Background reading
+
+[Nodes and Clients](https://ethereum.org/developers/docs/nodes-and-clients)
+
+#### Client implementations
+
+[Geth client](geth.ethereum.org)
```

Comment:
I think we should just link to here for list of execution client implementations: https://ethereum.org/en/developers/docs/nodes-and-clients/#execution-clients

This allows us to focus on only one place we ever need to keep things up to date.